website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,8 +20,24 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## June 2024
 
-- **New:** [Job warnings](/docs/deploy/job-notifications) are now GA. Previously, you could receive email or Slack alerts about your jobs when they succeeded, failed, or were canceled. Now with the new **Warns** option, you can also receive alerts when jobs have encountered warnings from tests or source freshness checks during their run. This gives you more flexibility on _when_ to be notified. 
+#### dbt Semantic Layer
+- **New:** Introduced a new granularity option for cumulative metrics, `period_agg`, nested under the `type_params` parameter. It specifies how to roll up the cumulative metric to another granularity. Options are `start`, `end`, `average`. Defaults to `start` if no `window` is specified.
+- **New:** Added validation that derived metrics must always have an `expr` key.
+- **New:** Enabled cache to pickup where filters that are included in saved queries.
+- **Enhancement:** [Cumulative metrics](/docs/build/cumulative) now support the `cumulative_type_params` key. The `cumulative_type_params` parameter replaces `type_params`. MetricFlow supports the old `type_params` key but will issue a warning if used.
+- **Enhancement:** Updated MetricFlow to use the new `cumulative_type_params` fields for cumulative metrics.
+- **Enhancement:** In [Google Sheets](/docs/cloud-integrations/semantic-layer/gsheets), we added information icons and descriptions to metrics and dimensions options in the Query Builder menu. Click on the **Info icon** button to view a description of the metric or dimension. Available in the following Query Builder menu sections: metric, group by, where, saved selections, and saved queries.
+- **Enhancement:** In [Google Sheets](/docs/cloud-integrations/semantic-layer/gsheets), you can now apply granularity to all time dimensions, not just metric time. This update uses our [APIs](/docs/dbt-cloud-apis/sl-api-overview) to support granularity selection on any chosen time dimension.
+- **Enhancement:** Improved querying error message when no semantic layer credentials were set.
+- **Fix:** Removed errors preventing querying cumulative metrics with other granularities.
+- **Fix:** Fixed various Tableau errors when querying certain metrics or when using calculated fields.
+- **Fix:** Relaxed naming field expectations to better identify calculated fields.
+- **Fix:** Fixed an error when refreshing database metadata for columns that we can't convert to Arrow. These columns will now be skipped. This mainly affected Hex users while refreshing the entire database metadata.
+- **Fix:** Fixed Private Link connections for Databricks.
+
```

Comment:
One more to add! 
- **New**: Added support for predicate pushdown SQL optimization. We will now push down categorical dimensions filters to the metric source table. Previously filters we're applied after we selected from the metric source table. This can help reduce full table scans on certain query engines.